src/main/kotlin/no/nav/syfo/dialogmote/api/v2/DialogmoteApiV2.kt

```diff
@@ -85,5 +89,39 @@ fun Route.registerDialogmoteApiV2(
                 call.respond(HttpStatusCode.OK)
             }
         }
+
+        get("/personident/motestatusendringer") {
+            val personident = getPersonIdentHeader()?.let { personident ->
+                PersonIdent(personident)
+            } ?: throw IllegalArgumentException("No Personident supplied")
+
+            validateVeilederAccess(
+                dialogmoteTilgangService = dialogmoteTilgangService,
+                personIdentToAccess = personident,
+                action = "GET dialogmote statusendringer for Personident"
+            ) {
+                val dialogmoteStatusEndringer = dialogmotestatusService.getMoteStatusEndringer(personident)
+
+                if (dialogmoteStatusEndringer.isEmpty()) {
+                    call.respond(HttpStatusCode.NoContent)
```

Comment:
Vil vi bruke `NoContent` eller bare returnere tom liste? Føler det ofte er enklere for frontend å forholde seg at til at responsen alltid er en liste.

Reply:
Blir data `undefined` ved 204? Trodde det bare ble en tom liste automatisk.

Reply:
Mener jeg har vært borti at det blir `undefined`, men det får dobbeltsjekkes.

Reply:
Nå returnerer jeg 204 og tom liste, var det noe sånt du tenkte?

Reply:
Tenkte egentlig bare respond med lista (ingen sjekk på empty eller bruk av `NoContent`)

Reply:
Jeg kan jo teste litt i dev og så se hvordan det oppfører seg 🤔